calico_mesos/calico_mesos.py

```diff
@@ -71,6 +72,19 @@ def calico_mesos():
     except KeyError:
         raise IsolatorException(ERROR_MISSING_ARGS)
 
+    # Unfortunately, netmodules passes in labels poorly, so move them
+    # around to match the agreed upon design.
+    if args.has_key('labels'):
+        # copy labels so we're not modifiying the original as we iterate
+        _labels = copy.deepcopy(args['labels'])
+        args['labels'] = {}
```

Comment:
Yes it shows the structure of args.... but here I think you need to show the structure of the overall message which includes a command.

Deffo need something about the format of the labels then on the input since you are converting it before calling the other functions.